standard/docs/en/support/governance.md

```diff
@@ -126,49 +128,49 @@ If there are at least two parties interested in using an extension, and followin
 
 ### Prioritization
 
-The technical team, with reference to community views, identify change proposals and extensions which should be considered for adoption in the next version of the standard, assigning these to milestones in the issue tracker where they are open for discussion. 
+The technical team, with reference to community views, identify change proposals and extensions which ought to be considered for adoption in the next version of the standard, assigning these to milestones in the issue tracker where they are open for discussion. 
 
 Periodically, at the start of a revision process a cut-off date for proposals will be announced  with at least two weeks notice. After that date a prioritized list of updates is produced. Any new proposed extensions or changes received after this period may not be considered until the next prioritization phase. 
 
 ### Prioritization review
 
 The list is shared for feedback, with at least a two-week window for discussion.
 
-Based on discussions, a final list is then proposed by the technical team with all the issues that will be taken forward into the rest of the process. A proposals that has made it this far may or may not make it into the final upgrade. As the proposal is worked into final concrete examples and schema changes further issues may arise.
+Based on discussions, a final list is then proposed by the technical team with all the issues that will be taken forward into the rest of the process. A proposal that has made it this far might or might not make it into the final upgrade. As the proposal is worked into final concrete examples and schema changes further issues can arise.
 
 ### Development & docs
 
 The technical team, working with community members, will work on a development branch to prepare updates to the schema, documentation and codelists, according to the prioritized list. 
 
 This stage is likely to involve broad community engagement and discussion of specific decisions through GitHub issues. 
 
-At the point where all open issues are suitably addressed, the development branch can be submitted for peer review.
+At the point where all open issues are suitably addressed, the development branch may be submitted for peer review.
```

Comment:
I don't think this should be "may" since the peer review step isn't optional. How about:

> At the point where all open issues are suitably addressed, the development branch **is ready to** be submitted for peer review.

Reply:
Changed to "should" to match following paragraph in 61f5cd6